src/test/java/org/ambraproject/rhino/service/impl/ArticleRevisionWriteServiceTest.java

```diff
@@ -111,24 +110,20 @@ public ArticleRevisionWriteService articleRevisionWriteService() {
    * Returns a mock query used to mock a HibernateTemplate.
    */
   private Answer<Query> createQueryAnswer() {
-    final Answer<Query> answer = new Answer<Query>() {
-      @Override
-      public Query answer(InvocationOnMock invocation) throws Throwable {
-        final String sql = invocation.getArgument(0);
-
-        // Return appropriate data based on SQL.
-        LOG.info("sql: {}", sql);
-        if (sql.contains("SELECT MAX(rev.revisionNumber)")) {
-          when(mockQuery.uniqueResult()).thenReturn(REVISION_NUMBER);
-        } else if (sql.contains("FROM ArticleIngestion")) {
-          when(mockQuery.uniqueResult()).thenReturn(expectedArticleIngestion);
-        } else if (sql.contains("FROM ArticleRevision")) {
-          when(mockQuery.uniqueResult()).thenReturn(expectedArticleRevision);
-        }
-        return mockQuery;
+    return invocation -> {
```

Comment:
Adding a little plug for IntelliJ here, which automatically created this lambda statement.